src/lib/pages/schedules.svelte

```diff
@@ -31,38 +39,65 @@
     'Create scheduled actions using our Public API or TCTL (CLI).';
 </script>
 
-<div class="flex flex-row justify-between">
+<div class="flex flex-col justify-between gap-2 md:flex-row">
   <div>
     <h1 class="flex items-center gap-2 text-2xl">
-      Schedules<Badge type="alpha">Alpha</Badge>
+      Schedules<Badge type="beta">Beta</Badge>
       <NamespaceSelector />
     </h1>
     <p class="text-sm text-gray-600" data-cy="namespace-name">
-      {namespaceName}
+      {namespace}
     </p>
   </div>
   <Button
     class="h-10"
     dataCy="create-schedule"
-    on:click={() => goto(routeForScheduleCreate({ namespace: namespaceName }))}
+    disabled={createDisabled}
+    on:click={() => goto(routeForScheduleCreate({ namespace }))}
     >Create Schedule</Button
   >
 </div>
-<div class="w-full xl:w-1/2">
-  <Input
-    icon="search"
-    id="schedule-name-filter"
-    placeholder="Schedule Name"
-    bind:value={search}
-    on:submit={noop}
-  />
-</div>
 
 {#await fetchSchedules}
   <Loading />
 {:then { schedules, error }}
   {#if schedules?.length}
     <Pagination items={filteredSchedules(schedules)} let:visibleItems>
+      <svelte:fragment slot="action-top-left">
+        <div class="w-full xl:w-1/2">
+          <Input
+            icon="search"
+            id="schedule-name-filter"
+            placeholder="Schedule Name"
+            clearable
+            bind:value={search}
+            on:submit={noop}
+          />
+        </div>
+      </svelte:fragment>
+      <svelte:fragment slot="action-top-right">
+        <DropdownButton
+          id="timezone"
+          label={capitalize($timeFormat)}
+          icon="clock"
+        >
+          <button
+            on:click={() => ($timeFormat = 'relative')}
+            class="timezone-label"
+          >
+            Relative
+          </button>
+          <button on:click={() => ($timeFormat = 'UTC')} class="timezone-label">
+            UTC
+          </button>
+          <button
+            on:click={() => ($timeFormat = 'local')}
+            class="timezone-label"
+          >
+            Local
+          </button>
```

Comment:
These should be `<MenuItems />` if possible.

Reply:
👍